snc.sh

```diff
@@ -44,7 +44,7 @@ if test -n "${OPENSHIFT_VERSION-}"; then
     OPENSHIFT_RELEASE_VERSION=${OPENSHIFT_VERSION}
     echo "Using release ${OPENSHIFT_RELEASE_VERSION} from OPENSHIFT_VERSION"
 else
-    OPENSHIFT_RELEASE_VERSION="$(curl -L "${MIRROR}"/candidate-4.11/release.txt | sed -n 's/^ *Version: *//p')"
+    OPENSHIFT_RELEASE_VERSION="$(curl -L "${MIRROR}"/candidate-4.12/release.txt | sed -n 's/^ *Version: *//p')"
```

Comment:
build-patched-kao-kcmo-images.sh uses latest-4.12, not sure if this is problematic?

Reply:
as of now there is no `latest-4.12` until the release so we are going to keep `candidate` till and switch to `latest` once available.

Reply:
Yes, and my point was that it's wrong for `build-patched-kao-kcmo-images.sh` to refer to `latest-${OCP_VERSION}` and that this should be changed to `candidate-${OCP_VERSION}`